src/mlpack/core/data/load_impl.hpp

```diff
@@ -410,6 +410,77 @@ bool LoadCategorical(const std::string& filename,
   return true;
 }
 
+template<typename MatType>
+bool Load(const std::vector<std::string>& filesname,
```

Comment:
```suggestion
bool Load(const std::vector<std::string>& filenames,
```
`filenames` instead of `filesname`? :smile: